### PR TITLE
updating the informaiton on two journal papers 2022 & 2023 from the NLST study

### PR DESCRIPTION
**Pull request by aeroelasticitylu**
2022 IEEE Access Paper
* Updated to the published version

2023 Heliyon Paper
* New info added

### CODE REVIEW

**Review by ashkanpakzad — CHANGES_REQUESTED** [review]
content/publication/lu2023_nlst_crnn/index.md

I believe it is because the abstract value has unmatched quotes causing the deploy to fail. Changing the value from single quotes ' to double quotes " should fix the problem.

**Comment by aeroelasticitylu** [comment]
I see I see. I missed that. Thx! Let me try again.

**Review by ashkanpakzad — APPROVED** [review]
